FLT/DedekindDomain/FiniteAdeleRing/BaseChange.lean

```diff
@@ -169,13 +169,28 @@ noncomputable local instance (w : HeightOneSpectrum B) :
     Algebra K (adicCompletion L w) := RingHom.toAlgebra <|
   (algebraMap L (adicCompletion L w)).comp (algebraMap K L)
 
+
 variable {B L} in
 noncomputable def adicCompletion_comap_algHom (w : HeightOneSpectrum B) :
-    letI : Algebra K (adicCompletion L w) := RingHom.toAlgebra <|
-  (algebraMap L (adicCompletion L w)).comp (algebraMap K L);
-    letI : Module K (adicCompletion L w) := Algebra.toModule
-    (HeightOneSpectrum.adicCompletion K (comap A w)) →ₐ[K] (HeightOneSpectrum.adicCompletion L w) :=
-  sorry -- use `adicCompletion_comap_ringHom` and prove it's a K-algebra homomorphism
+    (HeightOneSpectrum.adicCompletion K (comap A w)) →ₐ[K] (HeightOneSpectrum.adicCompletion L w) := by
+    use adicCompletion_comap_ringHom A K w
+    intro r
```

Comment:
```suggestion
    (HeightOneSpectrum.adicCompletion K (comap A w)) →ₐ[K]
    (HeightOneSpectrum.adicCompletion L w) where
  __ := adicCompletion_comap_ringHom A K w
  commutes' r := by
```
What you write compiles, but this is the idiomatic syntax. I also break the line which was over 100 characters.